mkdocs.yml

```diff
@@ -0,0 +1,41 @@
+site_name: "Scality S3 CSI Driver"
+site_url: https://scality.github.io/mountpoint-s3-csi-driver 
+repo_url:  https://github.com/scality/mountpoint-s3-csi-driver
+edit_uri:  edit/main/docs/          # ← branch that holds Markdown
```

Comment:
```suggestion
repo_url: https://github.com/scality/mountpoint-s3-csi-driver
edit_uri: edit/main/docs/          # ← branch that holds Markdown
```

Reply:
Thanks changed this, but need to review the whole PR again, as things changed I forced pushed everything